### PR TITLE
Integrate openSUSE to graphene_local_ci in gsc pipeline.

### PR DESCRIPTION
**Pull request by sreeharikax**

### CODE REVIEW

**Comment by sreeharikax** [comment]
http://ba5sapp0350:8080/job/harika_gsc_test_job/106/

File: gsc/test/suse-bash.dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ARG BUILD_OS
+
+From $BUILD_OS
+
+ARG BUILD_OS
+
+RUN if [ "$BUILD_OS" = "registry.suse.com/suse/sle15:15.4" ] || [ "$BUILD_OS" = "opensuse/leap:15.6" ]; then \
```
Comment (anjalirai-intel) [inline gsc/test/suse-bash.dockerfile:7]: If this file is only called for suse os, then why build os comparison

File: gsc/test/suse-helloworld.dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ARG BUILD_OS
+
+From $BUILD_OS
+
+ARG BUILD_OS
+
+RUN if [ "$BUILD_OS" = "registry.suse.com/suse/sle15:15.4" ] || [ "$BUILD_OS" = "opensuse/leap:15.6" ]; then \
```
Comment (anjalirai-intel) [inline gsc/test/suse-helloworld.dockerfile:7]: same above comment

File: pipelines/gramine_gsc_pipeline [hunk 3]
```diff
@@ -62,6 +62,18 @@ pipeline {
...
+
+                stage ('Invoke GSC with SUSE Linux Enterprise Server 15.4.') {
+                    steps {
+                        build job: 'local_ci_graphene_gsc', parameters: [[$class: 'NodeParameterValue', name: 'node_label', labels: ['graphene_22.04_6.7_rasp'], nodeEligibility: [$class: 'AllNodeEligibility']], [$class: 'StringParameterValue', name: 'distro_ver', value: "registry.suse.com/suse/sle15:15.4"]], propagate: true, wait: true
+                    }
+                }
+
+                stage ('Invoke GSC with openSUSE Leap 15.6') {
+                    steps {
+                        build job: 'local_ci_graphene_gsc', parameters: [[$class: 'NodeParameterValue', name: 'node_label', labels: ['graphene_22.04_6.7_rasp'], nodeEligibility: [$class: 'AllNodeEligibility']], [$class: 'StringParameterValue', name: 'distro_ver', value: "opensuse/leap:15.6"]], propagate: true, wait: true
```
Comment (anjalirai-intel) [inline pipelines/gramine_gsc_pipeline:74]: Assign it to some other system

File: ci/lib/stage-build-gsc.jenkinsfile [hunk 4]
```diff
@@ -71,8 +73,9 @@ stage('Build Workloads') {
 
     // GSC build-gramine Bash Workload
     try {
-        if (env.os_release_id.contains("centos") || env.os_release_id.contains("redhat")) {
+        if (env.os_release_id.contains("centos") || env.os_release_id.contains("redhat") || env.os_release_id.contains("suse") || env.os_release_id.contains("opensuse")) {
```
Comment (anjalirai-intel) [inline ci/lib/stage-build-gsc.jenkinsfile:76]: Can we simplify this condition

File: ci/lib/stage-build-gsc.jenkinsfile [hunk 5]
```diff
@@ -53,8 +54,9 @@ stage('Build Workloads') {
 
     // Build Bash HelloWorld Workload
     try {
-        if (env.os_release_id.contains("centos") || env.os_release_id.contains("redhat")) {
+        if (env.os_release_id.contains("centos") || env.os_release_id.contains("redhat") || env.os_release_id.contains("suse") || env.os_release_id.contains("opensuse")) {
```
Comment (anjalirai-intel) [inline ci/lib/stage-build-gsc.jenkinsfile:57]: Can we simplify this condition

File: ci/lib/stage-build-gsc.jenkinsfile [hunk 6]
```diff
@@ -8,8 +8,9 @@ stage('Build Workloads') {
 
     // Build Bash Workload
     try {
-        if (env.os_release_id.contains("centos") || env.os_release_id.contains("redhat")) {
+        if (env.os_release_id.contains("centos") || env.os_release_id.contains("redhat") || env.os_release_id.contains("suse") || env.os_release_id.contains("opensuse")) {
```
Comment (anjalirai-intel) [inline ci/lib/stage-build-gsc.jenkinsfile:11]: simplify this condition

File: gsc/test/suse-bash.dockerfile (hunk 1 above)
Comment (sreeharikax) [reply gsc/test/suse-bash.dockerfile]: Updated

File: gsc/test/suse-helloworld.dockerfile (hunk 2 above)
Comment (sreeharikax) [reply gsc/test/suse-helloworld.dockerfile]: Updated

File: pipelines/gramine_gsc_pipeline (hunk 3 above)
Comment (sreeharikax) [reply pipelines/gramine_gsc_pipeline]: Modified

File: ci/lib/stage-build-gsc.jenkinsfile (hunk 4 above)
Comment (sreeharikax) [reply ci/lib/stage-build-gsc.jenkinsfile]: Modified

**Comment by anjalirai-intel** [comment]
OpenSuse is integrated

**Pull request closed without merging**